connaisseur/tests/test_trust_data.py

```diff
@@ -260,7 +337,7 @@ def test_validate_schema_error(td, mock_schema_path):
         (trust_data("tests/data/sample_snapshot.json"), "snapshot"),
         (trust_data("tests/data/sample_timestamp.json"), "timestamp"),
         (trust_data("tests/data/sample_targets.json"), "targets"),
-        (trust_data("tests/data/sample_releases.json"), "targets/releases"),
+        (trust_data("tests/data/sample_releases.json"), "targets/releases")
```

Comment:
No need for this change. Also, you probably didn't run the Black code formatter; I think it puts in the comma?